src/components/share-modal/share-modal-component.jsx

```diff
@@ -59,15 +59,13 @@ const ShareModal = ({ handleClose, isOpen, shareSocialMedia }) => {
       </div>
       <div className={styles.socialMediaContainer}>
         {shareSocialMedia.map(socialMedia => (
-          <Button
+          <button
             onClick={() => window.open(`${socialMedia.link}${currentLocation}`)}
```

Comment:
We can avoid the lost of the params after the ampersand by encoding the URL before we send it over the to the given social media:
`onClick={() => window.open(`${socialMedia.link}${encodeURIComponent(currentLocation)}`)}`